src/api/mod.rs

```diff
@@ -207,8 +142,8 @@ pub struct Characteristic {
     pub properties: CharPropFlags,
 }
 
-impl Display for Characteristic {
-    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+impl fmt::Display for Characteristic {
```

Comment:
What's wrong with importing `Display`. `Debug`, `Formatter` and so on as we were before?

Reply:
That's just personal preference, rather use the module name than importing everything. Especially, as we have to use `fmt::Result` anyway.

However, if this is contrary to the common style there is no problem in importing stuff from `std::fmt` directly.